### DISCUSSION TITLE
One text over multiple codicological units

### DISCUSSION THREAD

**Discussion by MassimoVilla** [SChAth/ScAthiop#1805] [Q&A] [answered]
BGV009 is a composite MS containing the Vita and Miracles of ʾEwosṭātewos. It has three codicological units p1, p2, and p3.
A certain text (LIT3073RepCh353) begins in the last folios of p1 and continues on p2. This happened because another codicological unit (a simple bifolium) was added at the end of p1 but the scribe decided, for several reasons (textual continuity, to save space), to start writing from the first available leaf, that is at the end of p1.
How to deal with these cases? I tried to duplicate the textual information, and I've described the same text both as `<msItem xml:id="p1_i3">` (last text of p1) and as `<msItem xml:id="p2_i1">` (first and only text of p2), both with the same ID = LIT3073RepCh353, as follows:
```
 <msItem xml:id="p1_i3">
<locus from="75ra" to="75vb"/>
<title type="incomplete" ref="LIT3073RepCh353"/>
<textLang xml:lang="gez"/>
...
 <msItem xml:id="p2_i1">
<locus from="76ra" to="77ra"/>
<title type="incomplete" ref="LIT3073RepCh353"/>
<textLang xml:lang="gez"/>
```

Please note that I had to state `type="incomplete"` because each codicological unit contains only a part of the text, even though the text is overall complete and runs without interruption.
Also, what about incipits and explicits? The incipit of p1_i3 is the actual rubricated incipit, while the explicit of p2_i1 is the real conclusion of the text. This is undisputed. But the explicit of p1_i2 just coincides arbitrarily with the final lines of p1 (in this case a given word አንቀልቀላ። is split into the two parts), and the incipit of p2_i1 similarly coincides with the first lines at the beginning of p2. I encoded all these pieces of information, but I wonder whether giving the latter two portions of text is potentially misleading for the reader, who might erroneously believe that the text is incomplete or acephalous.

I do not know if my solution is the ideal one for such cases. In most cases textual units are simple submultiples of codicological units, so that the codicological subvidision forms the upper level and the textual subdivision the lower level, but here there's an overlap between the codicological and the textual sequences.

Also, how can this phenomenon be visualized in a printed output?

PS I am assigning also Thea for she is experienced in encoding and might have encountered a similar case

**Comment by PietroLiuzzo** [comment, marked as answer]
Dear @MassimoVilla , I would have done just like you did, and I would have explained perhaps the same you are writing here in the record itself. 
Additionally, because there is no assumption AFAIK that a msPart is defined on the basis of a material unit of production, you could have, for example a production unit, thus a msPart, which only contains your text and is defined on the basis of content. 
you would have, e.g. `msPart[@xml:id='p1']` with the `msItem[@xml:id='p1_i3']` as above, a `msPart[@xml:id='p2']` with the `msItem[@xml:id='p2_i1']` which is your complete text, this msPart not having further definition of support and a  `msPart[@xml:id='p3']` with the `msItem[@xml:id='p3_i1']` which lists the end of LIT3073RepCh353 as you have now. Two units defined on a material base, one defined by content. You could also use `ref[@type='msPart']` to relate these.
An additional set of structural `relation` elements could also be considered, to state what happens (addition of content and material) with respect to the first production unit. 
```xml
<relation active="BGV009#p1" name="sdc:constituteUnite" passive="BGV009#UniProd1"/>
<relation active="BGV009#p1" name="sdc:constituteUnite" passive="BGV009#UniMat1"/>
<relation active="BGV009#p2" name="sdc:constituteUnite" passive="BGV009#UniProd2"/>
<relation active="BGV009#p2" name="sdc:constituteUnite" passive="BGV009#UniCont1"/>
<relation active="BGV009#p3" name="sdc:constituteUnite" passive="BGV009#UniProd2"/>
<relation active="BGV009#p3" name="sdc:constituteUnite" passive="BGV009#UniMat2"/>
<relation active="BGV009#UniProd1" name="sdc:undergoesTransformation" passive="BGV009#tr1"/>
<relation active="BGV009#UniProd2" name="sdc:undergoesTransformation" passive="BGV009#tr1"/>
<relation active="BGV009#tr1" name="sdc:hasTransformationModel" passive="sdc:A2">
        <desc>This a transformation A2 Addition of material support and content.</desc>
    </relation>
```

In terms of presentation in print, this encoding can be used prioritizing content or units as desired, and can be made into a static rule. e.g. listing contents and grouping them by `title/@ref`.

**Comment by MassimoVilla** [comment]
Dear Pietro, thank you for your detailed reply. I will discuss the most convenient solution in terms of visualization in print (i.e. whether to prioritize content or units) together with the other members of our project. I am sure that even though we (at least I) do not appreciate now the full implications of the two options, everything will be clearer after a first try with the PDF package.

**Comment by PietroLiuzzo** [comment]
I think more than optional these encodings are incremental. you can simply do as you did, or add an additional msPart or add that and the relations, or only add the relations. It is all a matter of desired precision. for the PDF we will use what we have. If this encoding looks fine for you I would like to use it for an example in the Guidelines.

**Comment by MassimoVilla** [comment]
Of course you can use my example in the Guidelines and provide all the possible incremental encodings, i.e. just as I did, with the additional msPart and/or with the relations. I am inclined to add both the additional msPart and the relations in my record.
Still, I would like to discuss this issue together with Antonella and Susanne, so that for these instances we define a shared encoding internal to our project, just in case further similar cases are extant (a shared encoding will be helpful for the transformation into PDF as well). Our meeting is scheduled on next Tuesday.
